### PR TITLE
deploy raven testdata to thredds: switch to new repo raven-testdata

### PR DESCRIPTION
**Pull request by tlvu**
For automated raven testdata sync at https://pavics.ouranos.ca/twitcher/ows/proxy/thredds/catalog/birdhouse/testdata/raven/catalog.html

Will add the following new files from raven-testdata repo.  All existing are still there.
```
cd+++++++++ eccc_forecasts/                                                                                                                           
>f+++++++++ eccc_forecasts/geps_watershed.nc                                                                                                          
cd+++++++++ era5/                                               
>f+++++++++ era5/tas_pr_20180101-20180108.nc                                               
cd+++++++++ raven-routing-sample/                                                   
>f+++++++++ raven-routing-sample/VIC_streaminputs.nc
>f+++++++++ raven-routing-sample/era5-test-dataset-crop.nc
```

Part of issue https://github.com/Ouranosinc/raven/pull/324#pullrequestreview-540983876

### CODE REVIEW

**Comment by tlvu** [comment]
@huard sorry need approval, this is the repo that I can not merge without approval since it impacts production deployment.

**Review by huard — APPROVED** [review]

**Pull request merged**
